code/modules/client/preferences.dm

```diff
@@ -430,7 +430,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 
 			dat += "<b>Ghosts of Others:</b> <a href='?_src_=prefs;task=input;preference=ghostothers'>[button_name]</a><br>"
 
-			if (CONFIG_GET(flag/maprotation))
+			if (CONFIG_GET(number/maprotation_vote_method) == MAPVOTE_PREF)

```

Comment:
Allow people to set a pref even if the vote method is set to votes, use their pref if they don't vote, this way people can just set a pref and it will be used if they are too busy to cast a vote.